index.rst

```diff
@@ -46,12 +46,77 @@
+    * Data analysis and results from the data need to be easily related to when the data was taken.
+    * Search and find tasks and their results.
+
+Steps
+=====
+
+    * Run planning
+        * Which tasks/test are we running during a observation run?
+        * Night plans need to point to discrete tasks that are linked to where the procedure and artifacts are archived.
+          The plan then dictates required conditions and constraints on observations.
+        * We want to store these tasks/tests as JIRA Tickets.
+        * If using JIRA Tickets, we should use it as a place where we centralize relational information
+          (link to notebooks, LOVE scripts configuration files, test cases, confluence log pages, etc.)
+
+    * Nighttime observation
+        * Keep the Confluence Logging Page? Yes, for now it is the easiest place for nighttime staff to record task info.
+        * Should we add the Confluence Logging Page to the AuxTel Run ticket? Yes!
+        * We split the notebooks into task instead of having a single notebook for each observing night or observing run.
+          It seems that our team is happy with this new structure since it gives flexibility to run them in a different order it desired.
+          These notebooks now live in a private repository on GitHub, which is not ideal.
+          However, this new structure might give us a better idea about how do we want to organize them.
+
+    * Daytime JIRA compilation
+        * During the daytime, somebody must be resposible for extracting relevant information from nightlog and populating the associated JIRA ticket.
+
+    * Data Processing
+        * Do we want to have the data processing for each task case as a JIRA ticket?
+          And do we want to assign it to one or more person as a way of keeping track of which data was processed and how?
+          The processing tickets themselves will probably happen across many Jira projects.
+          We might have to adopt this workflow such that once the data is taken the ticket gets assigned to the person to handle the reduction/analysis (if obvious),
+          or it gets assigned to the run manager to handle the delegation.
+        * Data processing falls on the task proposer, the JIRA ticket should be assigned back to the task proposer
+          and/or the person who filled the ticket and results/analysis steps should be added to the JIRA ticket.
+        * The run manager, and all applicable personnel, should be tagged on the tickets.
+
+    * Results
+        * This is were we got major agreement:
+          we want to store the tests results (including planning, observing, processing) in to technotes (just like this one!).
+        * Some tasks may not get to this stage, but only require a short summary of results in the JIRA ticket.
+
+Example JIRA ticket Workflow
+============================
+
+   - Stakeholder develops idea for AuxTel nighttime task.
```

Comment:
What do think? Would it be interesting to have some sort of simple engineering task proposal platform? It'd be to mainly make sure all the information is contained and properly described when creating night notebooks or tasks.

Reply:
A "nighttime task" includes both science and engineering tasks.